docs/README.md

```diff
@@ -40,96 +33,57 @@ and considered to be a "beta" quality release. We expect significant
 
-Most users can install and start the Splunk Operator by just running
+A Kubernetes cluster administrator can install and start the Splunk Operator by running:
 ```
 kubectl apply -f https://github.com/splunk/splunk-operator/releases/download/0.2.2/splunk-operator-install.yaml
 ```
 
-Users of Red Hat OpenShift should read the additional
-[Red Hat OpenShift](OpenShift.md) documentation.
+The [Advanced Installation Instructions](Install.md) page offers guidance for advanced configurations, including the use of private image registries, installation at cluster scope, and installing the Splunk Operator as a user who is not a Kubernetes administrator. Users of Red Hat OpenShift should review the [Red Hat OpenShift](OpenShift.md) page.
 
-Please see the [Advanced Installation Instructions](Install.md) for
-special considerations, including the use of private image registries,
-installation at cluster scope, and installing as a regular user (who is
-not a Kubernetes cluster administrator).
+*Note: We recommended that the Splunk Enterprise Docker image is copied to a private registry, or directly onto your Kubernetes workers before creating large Splunk Enterprise deployments. See the [Required Images Documentation](Images.md) page, and the [Advanced Installation Instructions](Install.md) page for guidance on working with copies of the Docker images.*
 
-*Note: The `splunk/splunk:8.1.0` image (or later) is rather large, so we strongly
-recommend copying this to a private registry or directly onto your
-Kubernetes workers as per the [Required Images Documentation](Images.md),
-and following the [Advanced Installation Instructions](Install.md),
-before creating any large Splunk deployments.*
-
-After starting the Splunk Operator, you should see a single pod running
-within your current namespace:
+After the Splunk Operator starts, you'll see a single pod running within your current namespace:
 ```
 $ kubectl get pods
 NAME                               READY   STATUS    RESTARTS   AGE
 splunk-operator-75f5d4d85b-8pshn   1/1     Running   0          5s
 ```
 
-To remove all Splunk deployments and completely remove the
-Splunk Operator, run:
-```
-kubectl delete standalones --all
-kubectl delete licensemasters --all
-kubectl delete searchheadclusters --all
-kubectl delete clustermasters --all
-kubectl delete indexerclusters --all
-kubectl delete spark --all
-kubectl delete -f https://github.com/splunk/splunk-operator/releases/download/0.2.2/splunk-operator-install.yaml
-```
 
+## Creating a Splunk Enterprise deployment
+
+The `Standalone` custom resource is used to create a single instance deployment of Splunk Enterprise. For example:
 
-## Creating Splunk Enterprise Deployments
+1. Run the command to create a deployment named “s1”:
 
-The `Standalone` resource is used to create a single instance deployment
-of Splunk Enterprise. For example,  run the following command to create a
-deployment named “s1”:
 
 ```yaml
 cat <<EOF | kubectl apply -f -
```

Comment:
We'll want to update the apiVersion in the example to v1 per https://github.com/splunk/splunk-operator/pull/274